lib/transaction/auth_verification_step.js

```diff
@@ -57,14 +67,27 @@ authVerificationStep.prototype.verify = function verify(data) {
       verificationPayload = verificationPayload || {};
 
       if (err) {
+        if (acceptedCallback) {
+          events.ignoreError(self, err);
+
+          acceptedCallback(err);
+        }
+
         done(err);
         return;
       }
 
-      done(null, {
+      var payload = {
         // New recovery code if needed (recover)
         recoveryCode: verificationPayload.recoveryCode
-      });
+      };
+
+      // On the one hand we trigger the callback since the data has been
+      // accepted by the service provider, on the other hand
+      // we still need to wait for (loginOrRejectTask) to trigger the event.
+      // loginOrRejectTask might never be triggered in case of transport=manual
+      acceptedCallback(null, payload);
```

Comment:
Seems like this actually needs to be always there, so the previous `if` is not required ?